Resources/Prototypes/Entities/Structures/Piping/Atmospherics/portable.yml

```diff
@@ -94,13 +96,86 @@
                 min: 1
                 max: 3
               SheetGlass1:
+                min: 1
+                max: 2
+          - !type:DoActsBehavior
+            acts: [ "Destruction" ]
+
+- type: entity
+  id: SpaceHeater
+  parent: PortableAtmosMachine
```

Comment:
I think this works for the portable scrubber because "portable" is in the name and the sprite has wheels, but I was scratching my head for a while until I realized that this machine doesn't start anchored.

Reply:
I've always seen portable scrubbers, portable vents and space heaters as in the same category, but that's confusing indeed. Should i rename that to `unanchoredAtmosMachine` or something like that ?